doc/contribute/guidelines.rst

```diff
@@ -370,24 +370,35 @@ twister
 =======
 
 .. note::
-   twister does not currently run on Windows.
+   twister support on windows is limited and execution of tests is not
+   supported, only building.
 
 To verify that your changes did not break any tests or samples, please run the
-``twister`` script locally before submitting your pull request to GitHub. To
-run the same tests the CI system runs, follow these steps from within your
+``twister`` script locally before submitting your pull request to GitHub.
+
+Twister allowes limiting the scope of the tests built and run by pointing it to
+the tests related to the code or the platform you have modified.
+
+To run the same tests the CI system runs, follow these steps from within your
 local Zephyr source working directory:
 
 .. code-block:: console
 
     source zephyr-env.sh
-    ./scripts/twister
+    ./scripts/twister --integration
```

Comment:
What do you think of using `west twister` here instead of sourcing a shell script? Now that we have windows support, it might be worthwhile making the instructions work on windows too.